Makefile

```diff
@@ -19,7 +19,7 @@ deploy: setprivileges
 	kubectl create -f deploy/crd.yaml
 	kubectl create -f deploy/operator.yaml
 	# Create the operator resource (kubectl get apps)
-	kubectl create -f deploy/cr.yaml
+	# kubectl create -f deploy/cr.yaml - test only, try to have it failing just to see it travis catches it
```

Comment:
I would separate comment line with a code one. It's easier to uncomment and use it then.

Reply:
Right, but my aim here was was just to see if travis CI is really able to detect an issue that make the cdi-operator unable to deploy cdi.
We don't really want to merge this or reuse that comment.

Reply:
@tiraboschi yeah, me bline, read the title only after I submitted a comment, sorry.

Reply:
@alexxa the good news is that it worked as expected!